docs/failure-detector/failure-detector.md

```diff
@@ -0,0 +1,52 @@
+## _Failure detector_
+
+Main component in the failure detector is RemoteMonitoringService.
+The scheduler starts RemoteMonitoringService every 3 seconds (by default).
```

Comment:
Every 1 second.
Changed by #1845

Reply:
@xnull Please address all comments before merging your PR.

Reply:
maybe I shouldn't even tell a particular number,  it's an implementation specific thing.